File: app/src/main/java/com/frama/miserend/hu/utils/Validation.java
package com.frama.miserend.hu.utils;

/**
 * Created by Balazs on 2018. 03. 19..
 */

public class Validation {
    public static boolean isEmpty(String text) {
        return !notEmpty(text);
    }

    public static boolean notEmpty(String text) {
        return text != null && !text.isEmpty();
    }
}

File: app/src/main/java/com/frama/miserend/hu/utils/Validation.kt
package com.frama.miserend.hu.utils

/**
 * Created by Balazs on 2018. 03. 19..
 */

object Validation {
    fun isEmpty(text: String): Boolean {
        return !notEmpty(text)
    }

    fun notEmpty(text: String?): Boolean {
        return text != null && !text.isEmpty()
    }
}
